apps/prairielearn/src/lib/groups.ts

```diff
@@ -24,14 +24,6 @@ const GroupConfigSchema = z.object({
 });
 type GroupConfig = z.infer<typeof GroupConfigSchema>;
 
-const GroupMemberSchema = z.object({
-  uid: z.string(),
-  user_id: z.string(),
-  group_name: z.string(),
-  join_code: z.string(),
```

Comment:
I spot checked and I couldn't find a place where we actually accessed these properties of group members, so I think the replacement with the standard `User` schema is ok. I'd appreciate a double-check of this though.

Reply:
From what I checked I couldn't find a user of those either.